test/val/val_validation_state_test.cpp

```diff
@@ -58,7 +58,7 @@ TEST_F(ValidationStateTest, CheckNumGlobalVars) {
   )";
   CompileSuccessfully(spirv);
   EXPECT_EQ(SPV_SUCCESS, ValidateAndRetrieveValidationState());
-  EXPECT_EQ(2, vstate_->num_global_vars());
+  EXPECT_EQ(unsigned(2), vstate_->num_global_vars());
```

Comment:
This is ok.  I usually just use a "u" suffix.  It's short and sweet.
The compiler will do the upconversion to size_t, but at least 2u is of the right signedness type.

E.g. 2u